### PR TITLE
ls: refresh descriptions

### PR DESCRIPTION
**Pull request by Managor**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by Managor** [comment]
This was a quick and dirty edit. Probably could use even more adjustment.

Not sure how I should bracket the first line. [1] doesn't feel right but neither does [one].

File: pages/common/ls.md [hunk 1]
```diff
@@ -3,34 +3,34 @@
 > List directory contents.
 > More information: <https://www.gnu.org/software/coreutils/ls>.
 
-- List files one per line:
+- List files [one] per line:
```
[the remark this answers is not in the document]
Comment (Managor) [reply pages/common/ls.md]: Yeah... I'll leave it as is.

File: pages/common/ls.md [hunk 2]
```diff
@@ -7,30 +7,30 @@
...
 
 `ls -lh`
 
-- Long format list sorted by size (descending) recursively:
+- [l]ong format list sorted by [S]ize (descending) [R]ecursively:
 
 `ls -lSR`
 
-- Long format list of all files, sorted by modification date (oldest first):
+- [l]ong format list of files, sorted by [t]ime it was modified and in [r]everse order (oldest first):
```
Comment (spageektti) [inline pages/common/ls.md:30]:
```suggestion
- List files in [l]ong format, sorted by [t]ime the file was modified and in [r]everse order (oldest first):
```

File: pages/common/ls.md [hunk 3]
```diff
@@ -7,30 +7,30 @@
...
 
 `ls -la`
 
-- Long format list with size displayed using human-readable units (KiB, MiB, GiB):
+- [l]ong format list with size displayed using [h]uman-readable units (KiB, MiB, GiB):
 
 `ls -lh`
 
-- Long format list sorted by size (descending) recursively:
+- [l]ong format list sorted by [S]ize (descending) [R]ecursively:
```
Comment (spageektti) [inline pages/common/ls.md:26]:
```suggestion
- List files in [l]ong format, sorted by [S]ize (descending) [R]ecursively:
```

File: pages/common/ls.md [hunk 4]
```diff
@@ -7,30 +7,30 @@
...
 
 `ls -F`
 
-- Long format list (permissions, ownership, size, and modification date) of all files:
+- List [a]ll files in [l]ong format (permissions, ownership, size, and modification date):
 
 `ls -la`
 
-- Long format list with size displayed using human-readable units (KiB, MiB, GiB):
+- [l]ong format list with size displayed using [h]uman-readable units (KiB, MiB, GiB):
```
Comment (spageektti) [inline pages/common/ls.md:22]:
```suggestion
- List files in [l]ong format with size displayed using [h]uman-readable units (KiB, MiB, GiB):
```

**Review by spageektti — APPROVED** [review]

**Pull request merged**